aggregator/options.go

```diff
@@ -47,8 +47,8 @@ var (
 	defaultAggregationSumSuffix      = []byte(".sum")
 	defaultAggregationSumSqSuffix    = []byte(".sum_sq")
 	defaultAggregationMeanSuffix     = []byte(".mean")
-	defaultAggregationLowerSuffix    = []byte(".lower")
-	defaultAggregationUpperSuffix    = []byte(".upper")
+	defaultAggregationMinSuffix      = []byte(".min")
```

Comment:
I think it's actually ok, I feel lower or upper for a gauge is actually more confusing, I had to do lower and upper for timers for backwards compatibility but since we could decouple the suffix with the aggregation types, I feel we should support new metrics with less hacky suffixes